lib/token.coffee

```diff
@@ -118,6 +122,7 @@ exports.has = ->
 # token.remove()
 ###
 exports.remove = ->
+	delete process.env.RESIN_TOKEN
```

Comment:
I agree. In general `process.env` should be considered const. If you need to keep mutable state about some token you should initialise something else from the environment and then do any mutations there.